dataactbroker/fileRoutes.py

```diff
@@ -27,6 +27,32 @@ def add_file_routes(app, create_credentials, is_local, server_path):
     @requires_login
     def submit_files():
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
+
+        sess = GlobalDB.db().session
+
+        start_date = request.json.get('reporting_period_start_date')
```

Comment:
That's using a decorator that still utilizes `webargs`. It may be beneficial to utilize webargs the way other routes are using it. Not sure what the best practice is in this situation though.